packages/terra-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -79,10 +77,10 @@ describe('WorklistDataGrid', () => {
     );
 
     // Find column headers
-    const nonSelectableCell = wrapper.find(Row).at(0).find('th:not(.selectable)');
+    const nonSelectableCell = wrapper.find('th:not(.selectable)').first();
```

Comment:
We should have still be able to find the React components being rendered.  We would just have to use a string instead of using the import.  This could keep our tests pretty much the same.  I don't think we should be changing to using first and other concepts in this PR.  We just need to make sure everything is working.  Then we can look at minor updates.

Reply:
Reverted this and other changes in the test files. Only implement minimally needed changes.